CHANGELOG.md

```diff
@@ -36,6 +31,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Also, RPMs in the wild are "messy" and it is sadly commonplace for tags to present in the
   wrong header.
 
+### Breaking Changes
```

Comment:
Personal preference - I would prefer Breaking Changes to be the top thing in this section because they're the most important thing that a user reading a CHANGELOG would actually care about.

Reply:
Ok, done.